core.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+var time_interval = 500;
+var counter = 0, ok_value = 0, ng_value = 0;
+
+function changeFileName() {

```

Comment:
I think the name is misleading. Do you mean `getAndShowVerdict()` or so?

Reply:
yeah, I forgot to change that after change the method.